terraform/healthcheck/cloudwatch.tf

```diff
@@ -0,0 +1,30 @@
+# Adds permission for cloudwatch to invoke alerter function
+resource "aws_lambda_permission" "allow_cloudwatch_to_invoke_alerter" {
```

Comment:
Isn't this associated with the alerter functionality? Should it be in the healthcheck stuff?

Reply:
Yes because its also actions the sns topic and is dependent on the healthcheck lambda function. Therefore it needs to be in the healthcheck directory. We could put it top level and output the arn's but i think it should be with the healthcheck files